switch-base-branch.sh

```
@@ -0,0 +1,293 @@
+
+set -euo pipefail
+
+# ============================================================================
+# Colors and Logging
+# ============================================================================
+RED='\033[0;31m'
+GREEN='\033[0;32m'
+BGREEN='\033[1;32m'
+YELLOW='\033[1;33m'
+BLUE='\033[0;34m'
+GBLUE='\033[1;34m'
```

Shellcheck (0.11.0)

[warning] GBLUE appears unused. Verify use (or export if used externally).

(SC2034)